gguf-py/gguf/constants.py

```diff
@@ -693,6 +695,23 @@ class MODEL_TENSOR(IntEnum):
         MODEL_TENSOR.FFN_DOWN_EXP,
         MODEL_TENSOR.FFN_UP_EXP,
     ],
+    MODEL_ARCH.OLMO: [
+        MODEL_TENSOR.TOKEN_EMBD,
+        MODEL_TENSOR.OUTPUT,
+        MODEL_TENSOR.ROPE_FREQS,
+        MODEL_TENSOR.ATTN_Q,
```

Comment:
In the c code, you are supporting bias tensors but not in the maaping

Reply:
right, the c parts are adapted from the llama code. But since the OLMo architecture does not have them, they can be removed.